src/App.js

```diff
@@ -21,17 +31,21 @@ function App() {
       })
     );
     main.append(...playerElements);
+    nextPage = players.meta.next_page?.match(/\d+/)[0];
```

Comment:
You don´t need that regex, because, the pages are directly stored there, without an url you have to scrap, just try it without, mabye you need to test, if there is a value,
```suggestion
    nextPage = players.meta.next_page;
```